README.md

```diff
@@ -82,9 +82,9 @@
 
 There are several easy ways to get started with CopilotKit:
 
-- [**Quickstart: Chatbot:**](https://docs.copilotkit.ai/quickstart-chatbot?ref=github_readme) In just two minutes, add an AI Chatbot to your app with the ability to read application state and take actions.
+- [**Quickstart: Chatbot:**](https://docs.copilotkit.ai/quickstart) In just two minutes, add an AI Chatbot to your app with the ability to read application state and take actions.
```

Comment:
link should be - https://docs.copilotkit.ai/quickstart?ref=github_readme

Reply:
ok i will